ironfish/src/fileStores/config.ts

```diff
@@ -337,15 +337,15 @@ export class Config extends KeyStore<ConfigOptions> {
   }
 
   get chainDatabasePath(): string {
-    return this.files.join(this.storage.dataDir, 'databases', this.get('databaseName'))
+    return this.files.join(this.storage.dataDir, 'databases', 'chain', this.get('databaseName'))
   }
 
   get accountDatabasePath(): string {
-    return this.files.join(this.storage.dataDir, 'accounts', this.get('accountName'))
+    return this.files.join(this.storage.dataDir, 'databases', 'wallet', this.get('accountName'))
   }
 
   get indexDatabasePath(): string {
-    return this.files.join(this.storage.dataDir, 'indexes', this.get('databaseName'))
+    return this.files.join(this.storage.dataDir, 'databases', 'mined', this.get('databaseName'))
```

Comment:
We don't need the extra folder anymore now too.

We can remove the middle folder and just use the DB name since that becomes a folder

Reply:
also this should have it own DB name

Reply:
We can't do this with `accounts` unless we want to rename the default for the chain database to be not `default` or the default for accounts name to not be `default`, there is a collision. I left the folder path in there for accounts